docs/guides/js-waku/light-send-receive.md

```diff
@@ -100,17 +124,17 @@ To send messages over the Waku Network using the `Light Push` protocol, create a
 ```js
 // Create a new message object
 const protoMessage = ChatMessage.create({
```

Comment:
can we remote ChatMessage from the protobuf structure please? we can have a generic name like 'DataPacket' or something better so that we don't make it look like Waku is only for chat